bom-2.361.x/pom.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Conflicts with the changes that would be made by `updatecli/generate-manifests.ps1`, but that script has been disabled for quite some time and is effectively dead code that the maintainers ought to remove from this repository.